### DISCUSSION TITLE
connect kepserver “java.util.concurrent.CompletionException: UaServiceFaultException: status=Bad_Timeout, message=The operation timed out”

### DISCUSSION THREAD

**Discussion by cptjackwu** [eclipse-milo/milo#1055] [General]
1.When I connect to kepServer And use anonymous login，the following error occurs.
![1663573900417](https://user-images.githubusercontent.com/24241502/190972978-fc28d914-8395-4559-bcd9-83355f4d2429.jpg)
![1663573889671](https://user-images.githubusercontent.com/24241502/190973047-e2dd6186-b244-406e-a819-73580d23d053.jpg)

**Comment by kevinherron** [comment]
I'm not sure this is a client problem. Maybe you can get a Wireshark capture. It just looks like the server is reporting Bad_Timeout in response to some PublishRequest, which means it thinks it was going time out before it could use it.
